web/src/portico/team.ts

```diff
@@ -17,19 +17,30 @@ const all_repository_names = [
     "zulip-terminal",
     "zulip-ios-legacy",
     "zulip-android",
+    "zulip-flutter",
+] as const;
+
+const all_tab_names = [
+    "server",
+    "desktop",
+    "mobile",
+    "zulip-js",
+    "python-zulip-api",
+    "zulipbot",
+    "terminal",
 ] as const;
```

Comment:
Maybe we can compute these things as the set of keys and values in `tab_name_to_repo_list`? That would remove the need to enter each tab/repository multiple times.

Reply:
It's probably not super important.